dotCMS/src/main/java/com/dotmarketing/portlets/templates/action/EditTemplateAction.java

```diff
@@ -489,6 +489,11 @@ else if ((cmd != null)
 		// *********************** END GRAZIANO issue-12-dnd-template		
 	}
 
+	private boolean isDrawed(ActionRequest req) {
+		Boolean drawedParameter = req.getParameter("drawed") != null ? Boolean.valueOf(req.getParameter("drawed")) : null;
+		return drawedParameter == null && req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED) != null ? (Boolean)req.getAttribute(WebKeys.TEMPLATE_IS_DRAWED) : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 153 characters long line (which is greater than 150 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00103)